plugins/out_azure_logs_ingestion/azure_logs_ingestion.c

```diff
@@ -43,7 +43,6 @@ static int cb_azure_logs_ingestion_init(struct flb_output_instance *ins,
     /* Allocate and initialize a context from configuration */
     ctx = flb_az_li_ctx_create(ins, config);
     if (!ctx) {
-        flb_plg_error(ctx->ins, "configuration failed");
```

Comment:
Let's change this to `flb_plg_error(ins, "configuration failed");` which is what `ctx->ins` is set to inside `flb_az_li_ctx_create`.

Reply:
Thanks! Fixed!